scripts/fetch-poky.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e
+
+mkdir -p resources/poky
+cd resources/poky
+git clone https://github.com/yoctoproject/poky.git .
+git fetch --tags
+TMP_TAG=$(git tag | grep "yocto-" | tail -n 1) # There is a tag: yocto_1.5_M5.rc8 which will take the tail, thus adding a hyphen
```

Comment:
You should not use the latest tag here. Copy how fetch-docs work. Hardcode a specific tag commit. Then add the script to update it automatically through the github action, just like you added with the docs.

Reply:
Done.